core/services/run_manager.go

```diff
@@ -287,7 +287,7 @@ func (rm *runManager) ResumeAllPendingConnection() error {
 			logger.Errorw("Error saving run", run.ForLogger("error", err)...)
 		}
 	},
-		models.RunStatusPendingConnection)
+		models.RunStatusPendingConnection, models.RunStatusPendingOutgoingConfirmations)
```

Comment:
Does this belong in this PR?

Reply:
Yes, this wasn't covered by a test in one branch and so it was removed. In another branch we added a test for this behavior.